pkg/apis/v1alpha1/cluster_config_template_test.go

```diff
@@ -60,7 +60,8 @@ var _ = Describe("ClusterConfigTemplate", func() {
 				})
 
 				It("succeeds", func() {
-					Expect(template.ValidateCreate()).To(Succeed())
+					_, err := template.ValidateCreate()
```

Comment:
Did Gomega change all of a sudden? this has always worked.

Reply:
That is, the Gomega matchers for "Succeed" would match on the last element in a return

Reply:
controller-runtime changed their interface to return a set of warning in addition to the error. The change here is to handle a two value return instead of the previous single return.

Reply:
Yeah that's not what i was responding to, but I got my gomega matchers back to front. If you match on the first value, and the second value is of type `error` then the matcher will do automatic checking of err==nil for you.